source/simulator/core.cc

```diff
@@ -1365,6 +1403,95 @@ namespace aspect
 
 
   template <int dim>
+  class VectorFunctionFromVectorFunctionObject : public Function<dim>
```

Comment:
Ah, bummer. We probably want this class in deal.II too, then.

@rrgrove6 : Would you mind (once all questions about this class are settled) submitting this class as a patch to deal.II?